miyagi_trainer/dataloaders.py

```diff
@@ -1,25 +1,63 @@
 import os
 import inspect
-from collections import defaultdict, Counter
+from collections import Counter
 
 import torch
 import torchvision
 import torchvision.transforms as transforms
 
+from datasets import CUSTOM_DATASETS
 
 
-def _get_pytorch_dataloders(dataset, batch_size, num_workers, balanced_weights = False):
+def _get_pytorch_dataloders(
+        dataset, batch_size, num_workers, balanced_weights=False,
+        multiple_datasets_temperature=0.0):
+    
     if balanced_weights:
```

Comment:
I know that it was me that did that, but I forgot, if I set balanced_weights the samples will be balanced by the size of the class, that's it? What you implemented is a way that you can give more/less importance to other sources of data, datasets, right?

Additionally, you cannot combine both, right?

Maybe we should change the balanced_weights parameter's name to be more descriptive.